package-lock.json

```diff
@@ -1820,7 +1820,7 @@
     },
     "duplexer": {
       "version": "0.1.1",
-      "resolved": "https://registry.npmjs.org/duplexer/-/duplexer-0.1.1.tgz",
+      "resolved": "http://registry.npmjs.org/duplexer/-/duplexer-0.1.1.tgz",
```

Comment:
Can you please remove these erroneous changes? (`https` &rarr; `http`)

Reply:
good catch; not sure what happened there, but will get on it!

Reply:
Likely [this ticket](https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285/40).